src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -171,4 +178,22 @@ public void waitForVoteCountChangeTimeLagToPass() {
       e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1148)